primitives/src/parachain.rs

```diff
@@ -88,6 +88,17 @@ application_crypto::with_pair! {
 /// so we define it to be the same type as `SessionKey`. In the future it may have different crypto.
 pub type ValidatorSignature = validator_app::Signature;
 
+/// The key type ID for a fisherman key.
+pub const FISHERMAN_KEY_TYPE_ID: KeyTypeId = KeyTypeId(*b"fish");
```

Comment:
I think it's OK to have this key type, as the one in the Grandpa equivocations PR is in substrate-node, not a common library.

@andresilva does this conflict with #1000 ?

Reply:
We will use this key in polkadot for all equivocation reporting across BABE, GRANDPA and Parachains. It conflicts with #1000 since I added the same key, but I'll change it after this one is merged.